front/app/containers/Admin/projects/project/analysis/Insights/index.tsx

```diff
@@ -21,7 +21,7 @@ const Insights = () => {
 
   return (
     <Box>
-      <Box display="flex" gap="4px">
+      <Box>
```

Comment:
Worked on this a bit in https://github.com/CitizenLabDotCo/citizenlab/pull/5680 , we'll have a merge conflict

Reply:
Yes, I saw but that's ok, it's pretty minor.